modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/notification-templates/next-steps-template-paid-or-trial/en-US.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta content="IE=edge" http-equiv="X-UA-Compatible" />
+		<meta content="width=device-width, initial-scale=1.0" name="viewport" />
+
+		<title>DXP App Template</title>
```

Comment:
Is this title correct? I am asking this because it has the same name than the other template.

Reply:
See the comment above.